deploy/eck-operator/values.yaml

```diff
@@ -184,9 +186,15 @@ config:
   # certificatesRotateBefore defines when to rotate a certificate that is due to expire.
   certificatesRotateBefore: 24h
 
+  # disableConfigWatch specifies whether the operator watches the configuration file for changes.
+  disableConfigWatch: false
+
   # exposedNodeLabels is an array of regular expressions of node labels which are allowed to be copied as annotations on Elasticsearch Pods.
   exposedNodeLabels: [ "topology.kubernetes.io/.*", "failure-domain.beta.kubernetes.io/.*" ]
 
+  # ipFamily specifies the IP family to use. Possible values: IPv4, IPv6 and "" (auto-detect)
+  ipFamily: ""
```

Comment:
The odd little thing is that it looks like we are setting `auto-detect` by default here, which we'll actually do, but not by relying on `ip-family: ""` in the configMap but rather via the default value set in [main.go](https://github.com/elastic/cloud-on-k8s/blob/20f11e756989d9811d3f22f638949a81409a4ef6/cmd/manager/main.go#L260-L263
).
But this is probably better than writing `ip-family:` empty in the configMap (by removing the `with` directive) or leave this commented.

Reply:
Yeah, I initially had `ip-family` set without a `with` directive, and changed it upon seeing how it was being represented in the configmap iirc.